docs/graphs/classdiagram_lib.dot

```diff
@@ -0,0 +1,34 @@
+digraph hierarchy {
+size="5,5"
+node[   shape=record,
+        style=filled,
+        fillcolor=gray95]
+edge[dir=back, arrowtail=none]
+
+
+ps[label = "{ProgramState|
+                all_transactions\l
```

Comment:
Does this contain transactions of different bank accounts/credit cards/...? If so, how can we distinguish transactions for different bank accounts from one another?

Reply:
I would just store it in one flat table, and add a column to indicate where it comes from

Reply:
Groupimg by that column is easier than merging, I think